packs/actions/sustain-an-effect.json

```diff
@@ -0,0 +1,30 @@
+{
+    "_id": "UqUDWO0TuSzEAKpw",
+    "img": "systems/pf2e/icons/actions/Passive.webp",
+    "name": "Sustain an Effect",
+    "system": {
+        "actionType": {
+            "value": "passive"
+        },
+        "actions": {
+            "value": null
+        },
+        "category": "interaction",
+        "description": {
+            "value": "<p>You Sustain one effect with a sustained duration while moving at half speed. Most such effects can be sustained for 10 minutes, though some specify they can be sustained for a different duration. Sustaining an effect that requires making complex decisions, such as spectral weapon, can make you @UUID[Compendium.pf2e.conditionitems.Item.Fatigued], as determined by the GM.</p>"
```

Comment:
Consider linking or emphasizing _spectral weapon_.

Reply:
Added emphasis to spectral weapon.